### ISSUE TITLE
Can MonitorControl Switch the Monitor Input Select?

### ISSUE THREAD

**Issue by waldobuck** [MonitorControl/MonitorControl#1343]
### Before opening the issue, have you...?

- [X] Searched for existing issues

### Question

I have a Viewsonic VP3256-4K 32” flat screen monitor.

Does MonitorControl have any functionality to change the input select?

Or does anyone know if there is software to set a keyboard shortcut that I can use to switch the input source from my MacBook Pro to my Windows PC without having to go through the process of pushing the 4 button combination on the back of the monitor?

I switch the input source numerous times a day and this is becoming a major inconvenience.

**Comment by waydabber** [comment]
Yes, [BetterDisplay](https://betterdisplay.pro) can do that (it's a free feature). After installing, just expand the display's menu and use the `DDC Input Source` submenu.

<img width="537" alt="Screenshot 2023-03-20 at 22 21 39" src="https://user-images.githubusercontent.com/37590873/226468757-25355912-7b1c-4288-8cf4-f150229d677b.png">

You can customize the list of inputs and its assignments under app Settings (gear icon)/Displays.
